### ISSUE TITLE
Support non-Latin characters in AMP Story Quizzes

### ISSUE THREAD

**Issue by jackbsteinberg** [ampproject/amphtml#25716]
As #25495 makes its way into AMP the font only supports Latin characters, and will need to be updated with proper i18n for non-Latin character sets as the project progresses.

Ties in with the i18n milestone in #25615

/cc @gmajoulet @newmuis

**Comment by gmajoulet** [comment]
@mszylkowski @processprocess 

cc @ampproject/wg-stories

**Comment by mszylkowski** [comment]
I think the main issue is the lack of support for non-latin characters is Poppins, the font used for the text. The fallback is currently sans-serif (and it doesn't look bad in the few languages I tested), but we can set another fallback for non-latin. However, someone that understands non-latin languages should choose a font in that case. Would it help if I set up a story with quizzes in many languages and post the screenshots here?

**Comment by gmajoulet** [comment]
Yes feel free to ask @hongwei1990 for guidance on what we should do here :))
